scripts/local_env_create_vm

```diff
@@ -0,0 +1,118 @@
+    mkdir -p "$CIFMW_LOCAL_ENV_IMAGES_DIR"
+    curl -l -o "$CIFMW_LOCAL_ENV_BACKING_IMG" "${CIFMW_LOCAL_ENV_BACKING_IMG_URL}"
+    curl -l -o "${CIFMW_LOCAL_ENV_BACKING_IMG}.SHA256SUM" "${CIFMW_LOCAL_ENV_BACKING_IMG_URL}.SHA256SUM"
+    (cd "$CIFMW_LOCAL_ENV_IMAGES_DIR" ; sha256sum -c "${CIFMW_LOCAL_ENV_BACKING_IMG}.SHA256SUM")
+fi
+
+rm -rf "$CIFMW_LOCAL_ENV_VM_IMG"
+
+qemu-img create -o backing_file="$CIFMW_LOCAL_ENV_BACKING_IMG",backing_fmt=qcow2 -f qcow2 "$CIFMW_LOCAL_ENV_VM_IMG" "120G"
+virt-sysprep -a "$CIFMW_LOCAL_ENV_VM_IMG" --network \
+  --ssh-inject root:file:${CIFMW_LOCAL_ENV_CALLING_USER_HOME}/.ssh/id_rsa.pub \
+  --selinux-relabel \
+  --root-password "password:fooBar" \
+  --firstboot "${SCRIPTPATH}/local_env_vm_first_boot" \
+  --update \
+  --write "/root/pull-secret:$(cat ${CIFMW_LOCAL_ENV_PULL_SECRET_PATH})" \
+  --install vim,git,tmux,libvirt,libvirt-client,libvirt,libvirt-daemon-kvm,virt-install,qemu-kvm,make,bash-completion,guestfs-tools,libguestfs,python3-pip
+
+virt-install --import \
+--name "$CIFMW_LOCAL_ENV_VM_NAME" \
+--memory 22572 \
+--vcpus 6 \
+--cpu host \
+--os-variant=centos-stream9 \
+--graphics spice \
+--noautoconsole \
+--network=default \
+--disk "$CIFMW_LOCAL_ENV_VM_IMG"
+
+CIFMW_LOCAL_ENV_VM_IP_CNT=0
+echo "Waiting for VM to come up"
+until [[ "$CIFMW_LOCAL_ENV_VM_IP_CNT" -ge 5 ]]
+do
+    CIFMW_LOCAL_ENV_VM_IP=$(virsh -q domifaddr --domain "$CIFMW_LOCAL_ENV_VM_NAME" | awk -F ' ' '{print $4}' | cut -d "/" -f 1)
+    if [[ ! -z $CIFMW_LOCAL_ENV_VM_IP ]]; then
+        break
+    fi
+    CIFMW_LOCAL_ENV_VM_IP_CNT=$((CIFMW_LOCAL_ENV_VM_IP_CNT+1))
+    sleep 5
+done
+
+if [ -z "$CIFMW_LOCAL_ENV_BACKING_IMG" ]; then
+    echo "Unable to find the $CIFMW_LOCAL_ENV_VM_NAME IP"
+    exit 1
+fi
+
+# Copy local ci-framework to target
+rsync -r -l \
+-e "ssh -i ${CIFMW_LOCAL_ENV_CALLING_USER_HOME}/.ssh/id_rsa -o StrictHostKeyChecking=no" \
+"$SCRIPTPATH/../" \
+"zuul@$CIFMW_LOCAL_ENV_VM_IP:/home/zuul/src/ci-framework"
+
+echo "Running post install"
+# Note: Seems like selinux is blocking libvirt related packages installed using the "--firstboot" option
+#    of virt-sysprep. This hack allows installing all the needed tools by:
+#    1) Split the post installation in two:
+#        - A virt-sysprep postinstall called local_env_vm_first_boot that creates the zuul user and gives him sudo permissions
+#        - A ssh executed script that install all the need dependencies
+#    2) Install prepare the VM image and install it
+#    3) If install succeeds, run the post-installation script through ssh
+ssh \
+-o StrictHostKeyChecking=no \
+-i "${CIFMW_LOCAL_ENV_CALLING_USER_HOME}/.ssh/id_rsa" \
+"zuul@$CIFMW_LOCAL_ENV_VM_IP" \
+"$(< ${SCRIPTPATH}/local_env_vm_prepare)"
+
+if [[ $? -ne 0 ]]; then
+    echo "Post installation script failed"
+    exit $?
+fi
+
+echo "Local environment setup success! VM IP: $CIFMW_LOCAL_ENV_VM_IP. Use 'ssh zuul@$CIFMW_LOCAL_ENV_VM_IP' to connect"
```

Comment:
Can we add one more line to guide the users for further steps.
```
echo " Run deploy-edpm.yml to deploy openstack: cd ~/src/ci-framework; ansible-playbook deploy-edpm.yml -e @scenarios/centos-9/local-env.yml"
```

Reply:
@raukadah  We also now have that documented on the official wiki here:

https://github.com/openstack-k8s-operators/ci-framework/wiki/Quickstart

Reply:
New role has something like that - and we'll have to update the doc accordingly as well.